staging/src/k8s.io/apiserver/pkg/storage/value/transformer.go

```diff
@@ -151,6 +152,11 @@ func (t *prefixTransformers) TransformFromStorage(ctx context.Context, data []by
 			// Thus we never short-circuit on a prefix match that results in an error.
 			if err != nil {
 				errs = append(errs, err)
+				klog.VDepth(2, 2).ErrorS(
+					err,
+					"failed to decrypt data",
+					"prefix", string(transformer.Prefix),
```

Comment:
we should log the resource associated with the call as well

Reply:
To log the data to be encrypted/decrypted is meant to help with debugging. but this could be a security/privacy concerns @enj WDYT? If so, then we shouldn't log it.

Reply:
We might end up logging plain text data in Encryption which might be sensitive and some obfuscated value in Decrypt.

Reply:
@ritazh, I am now logging the request info based on my discussion with @enj. PTAL when you get a chance.